src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx

```diff
@@ -1,19 +1,15 @@
-import React from "react";
 import { Button } from "@/common/components/atoms/button";
-import { FaArrowLeft } from "react-icons/fa6";
-import { RiExternalLinkLine } from "react-icons/ri";
 import { Progress } from "@/common/components/atoms/progress";
-import type { ProposalData } from "@/fidgets/community/nouns-dao";
-import {
-  StatusBadge,
-  estimateBlockTime,
-} from "@/fidgets/community/nouns-dao/components/ProposalListView";
-import { mergeClasses } from "@/common/lib/utils/mergeClasses";
 import Spinner from "@/common/components/atoms/spinner";
+import { mergeClasses } from "@/common/lib/utils/mergeClasses";
+import type { ProposalData } from "@/fidgets/community/nouns-dao";
 import moment from "moment";
+import { FaArrowLeft } from "react-icons/fa6";
+import { RiExternalLinkLine } from "react-icons/ri";
 import { useEnsName } from "wagmi";
 import { mainnet } from "wagmi/chains";
-
+import { StatusBadge, estimateBlockTime } from "./proposalListRowItem";
+import React from "react";
```

Comment:
Again, why the re-ordering of imports

Reply:
same as this: https://github.com/Nounspace/nounspace.ts/pull/176/files#r1661673126